airflow/migrations/versions/a84a5abfca95_update_dagrun_state_and_datetime.py

```diff
@@ -0,0 +1,70 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""update-dagrun-state-and-datetime
+
+Revision ID: a84a5abfca95
+Revises: 30867afad44a
+Create Date: 2021-06-13 11:08:18.705168
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+from airflow.utils.dates import timezone
+from airflow.utils.state import State
+
+# revision identifiers, used by Alembic.
+revision = 'a84a5abfca95'
+down_revision = '30867afad44a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Apply Set default start_time for dagrun to None and set default state to QUEUED"""
+    with op.batch_alter_table('dag_run') as batch_op:
+        batch_op.alter_column(
+            'start_date',
+            type_=sa.TIMESTAMP(timezone=True),
+            existing_server_default=timezone.utcnow(),
```

Comment:
We don't have any `server_default`, we just had a `default`:

https://docs.sqlalchemy.org/en/14/core/defaults.html#scalar-defaults

https://docs.sqlalchemy.org/en/14/core/metadata.html#sqlalchemy.schema.Column.params.server_default
https://docs.sqlalchemy.org/en/14/core/metadata.html#sqlalchemy.schema.Column.params.default

Reply:
![image](https://user-images.githubusercontent.com/8811558/123711038-b08a0b00-d867-11eb-9fd8-60565abfd999.png)

https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.add_column